hpOneView/common.py

```diff
@@ -1695,6 +1696,24 @@ def make_alertMap_dict(notes, etag, state='Active', user='None',
     }
 
 
+def make_migration_information(oaIpAddress, oaUsername, oaPassword, vcmUsername, vcmPassword, iloLicenseType='OneView',
```

Comment:
Or, just a global method outside the VcMigrationManager class but still inside vcmigrationmanager.py?